javascript/sdk/lib/types.ts

```diff
@@ -194,15 +194,15 @@ export interface Rule {
   description: string;
 }
 
-export interface RuleUsage {
+export interface UnitUsage {
   tests: string[];
   count: number;
   failed: number;
 }
 
 export interface Unit {
   rule: Rule | null;
```

Comment:
Rule :)

Reply:
wait but I just made the change above like you said RuleUsage -> UnitUsage